lib/Form/Block/Base.php

```diff
@@ -4,6 +4,7 @@
 
 use MailPoet\Form\Util\FieldNameObfuscator;
 use MailPoet\Models\ModelValidator;
+use MailPoet\WP\Functions as WPFunctions;
```

Comment:
What happened here? Was it broken before?

Reply:
Yes, my script didn't handle the `__` function well.